### ISSUE TITLE
Idea for the report table: A mini-plot that shows where Ns are in the assemblies

### ISSUE THREAD

**Issue by RaverJay** [replikation/poreCov#179]
Could also include warnings when N stretches are in important regions, e.g. the spike protein

As any mutations/deletions/insertions are just omitted when the corresponding region isn't covered enough, this might give a better overview what to expect.

Not sure on the details - and no idea where to find the time :D

**Comment by hoelzer** [comment]
:) good suggestion, actually we have something in this direction in place in covPipe, @MarieLataretu implemented a process that checks for important mutations and alerts when they are masked by N. 

For example, S:N501Y and when here an amplicon drop happens the user will get informed somehow. Am I right, @MarieLataretu ?

**Comment by replikation** [comment]
status? @MarieLataretu @RaverJay @hoelzer

**Comment by RaverJay** [comment]
No time, so i guess this remains in the idea stage :<

**Comment by MarieLataretu** [comment]
> For example, S:N501Y and when here an amplicon drop happens the user will get informed somehow. Am I right, @MarieLataretu ?

You can specify a VCF file - the process checks if the position is N/low coverage and if the variation is found exactly as in the VCF or with another ALT. The output is a hand full of VCF files which are transformed into a table for the summary report in covpipe.

I cloud plug this also into poreCov, but we still have to think about the output

**Comment by replikation** [comment]
sounds good. so it will be in the final report then?

**Comment by MarieLataretu** [comment]
Yes, with the current implementation, they just appear in the HTML report, which makes it a bit long:

![report](https://user-images.githubusercontent.com/52002068/204793950-a75d3b27-b401-45d9-9b95-a6274326afc6.PNG)

**Comment by hoelzer** [comment]
I like the Spike-focused plots bc/ they are pretty helpful to briefly check for "masked" regions. 

Can we just have a "box" that one can extend to look at these plots? So the normal report does not get longer?

**Comment by replikation** [comment]
+1 for the hide part

**Comment by MarieLataretu** [comment]
I'll see what I can do :)

You are thinking of the collapsibles we already have in the HTML table, right?

**Comment by hoelzer** [comment]
Yeah, something like a button below the genome.coverage plots to expand all the spike zoomed plots

**Comment by MarieLataretu** [comment]
#### Default

![grafik](https://user-images.githubusercontent.com/52002068/204837872-2d411a5c-3ecf-4ea4-a77a-d7986ec53121.png)

[bottom of the site]

---

#### Expanded

![grafik](https://user-images.githubusercontent.com/52002068/204838028-00eb10b1-0070-4b22-b826-9ff94abbb1db.png)

**Comment by replikation** [comment]
great love it
